app/impact-reports.server.ts

```diff
@@ -17,13 +17,20 @@ let hypercertClient: HypercertClient | null = null;
  * @returns A promise that resolves to an array of reports.
  * @throws Throws an error if fetching reports fails.
  */
-export const fetchReports = async (ownerAddress: string): Promise<Report[]> => {
+export const fetchReports = async (): Promise<Report[]> => {
+	if (!process.env.HC_OWNER_ADDRESS) {
+		throw new Error("Owner address environment variable is not set");
+	}
+	const ownerAddress = process.env.HC_OWNER_ADDRESS;
+
 	try {
 		if (reports) {
 			console.log("Reports already exist, no need to fetch from remote");
 			console.log(`Existing reports: ${reports.length}`);
 		} else {
-			console.log("Fetching reports from remote");
+			console.log(
```

Comment:
don't think we should be leaking the owner address via logs

Reply:
hmm, it's only showing up on the server side, so I don't think it'll be a problem, right?

Reply:
Its registering to me as an attack vector for the app

>  so I don't think it'll be a problem

lol sounds like someone who's about to be hacked but doesn't know it yet

Reply:
> Its registering to me as an attack vector for the app
> 
> > so I don't think it'll be a problem
> 
> lol sounds like someone who's about to be hacked but doesn't know it yet

I might be missing something here, but accessing the server essentially grants visibility into the source code and any embedded values. And address is not a sensitive information. But if you strongly feel it should be removed, I'm willing to do so

Reply:
well if you dont think so, leggo